x-pack/plugins/beats_management/server/lib/adapters/beats/elasticsearch_beats_adapter.ts

```diff
@@ -8,17 +8,17 @@ import { flatten, get as _get, omit } from 'lodash';
 import { INDEX_NAMES } from '../../../../common/constants';
 import { CMBeat } from '../../../../common/domain_types';
 import { DatabaseAdapter } from '../database/adapter_types';
+
 import { BackendFrameworkAdapter } from '../framework/adapter_types';
 import { FrameworkUser } from '../framework/adapter_types';
 import { BeatsTagAssignment, CMBeatsAdapter } from './adapter_types';
 
 export class ElasticsearchBeatsAdapter implements CMBeatsAdapter {
   private database: DatabaseAdapter;
-  private framework: BackendFrameworkAdapter;
```

Comment:
Does the inclusion of the `user: FrameworkUser` parameters for these endpoints make the `framework` field obsolete?

Reply:
fixed